docs/Operators.md

```diff
@@ -3785,6 +3890,26 @@ opset_import {
 </dl>
 
 
+#### Examples
+
+<details>
+<summary>reciprocal</summary>
+
+```python
+node = onnx.helper.make_node(
+    'Reciprocal',
+    inputs=['x'],
+    outputs=['y'],
+)
+x = np.random.randn(3, 4, 5).astype(np.float32)
```

Comment:
Probably `+ 2` to go away from 0?

Reply:
randn is normal distribution so there is still chance x < -2. 
I added a line to avoid element with value 0:
    x[x==0] = 1.

Reply:
Elements close to zero will have issues too. Use `np.random.rand(...) + 1`?